### PR TITLE
Using function arguments

### PR DESCRIPTION
**Pull request by karanchahal**
Using function argument instead of hard coding output activation in function. Previously there was a None insead of the output_activation variable. No change in functionality. Just found this while perusing through the code :)

### CODE REVIEW

**Comment by jachiam** [comment]
Hello @karanchahal! Thanks for thinking of this, but the original is set up as-intended. This function gets an `output_activation` argument so that it has the same signature as `mlp_gaussian_policy`, but should not use an output activation on the logits. The logits should have range -inf to inf, so that the policy can express the full range of possible probability distributions (since the probability of actions is based on the softmax of the logits). Let me know if you have any other questions about this!
